edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/Authenticator.cs

```diff
@@ -23,15 +24,9 @@ public Authenticator(IAuthenticator tokenAuthenticator, string edgeDeviceId, ICr
             this.credentialsCache = Preconditions.CheckNotNull(credentialsCache, nameof(ICredentialsCache));
         }
 
-        /// <summary>
-        /// Authenticates the client credentials
```

Comment:
Why are these comments being removed? They were added deliberately..

Reply:
Method name is already self-descriptive, therefore I removed it.  Will revert these changes.

Reply:
That is fair.. no need to revert this.